control_panel_api/settings/base.py

```diff
@@ -124,5 +124,6 @@
 LOGS_BUCKET_NAME = os.environ.get('LOGS_BUCKET_NAME', 'moj-analytics-s3-logs')
 IAM_ARN_BASE = os.environ.get('IAM_ARN_BASE', '')
 K8S_WORKER_ROLE_NAME = os.environ.get('K8S_WORKER_ROLE_NAME', '')
-
+SAML_PROVIDER_ARN = os.environ.get('SAML_PROVIDER_ARN', '')
```

Comment:
Don't forget to update [`cpanel` helm chart](https://github.com/ministryofjustice/analytics-platform-helm-charts/tree/master/charts/cpanel) and `cpanel`'s [dev](https://github.com/ministryofjustice/analytics-platform-config/blob/master/chart-env-config/dev/cpanel.yml) and [alpha](https://github.com/ministryofjustice/analytics-platform-config/blob/master/chart-env-config/alpha/cpanel.yml) configs or this would break the app when deployed.

(Ask me or @andyhd, and probably Kerin when you get to that point)